tipsy-bot/README.md

```diff
@@ -0,0 +1,24 @@
+# Tipsy Bot
+
+A drink server robot using Viam platform. [Learn how it's built!](https://docs.viam.com/tutorials/projects/tipsy/)
+
+## Getting Started
+
+0. Upload the `effdet0.tflite` model and the `labels.txt` file [in the app](https://app.viam.com/data/models), and set it up as an mlmodel and vision detector under the robot configuration
+0. Set environment variables (see note below)
+0. run `pip3 install -r requirements.txt` in this directory
+0. run `python tipsy.py`
+
+**Required environment variables:**
+
+[Learn about authenticating with the Viam SDK](https://docs.viam.com/program/run/#authentication).
+
+- `ROBOT_SECRET`
+- `ROBOT_ADDRESS`
+
+**Optional environment variables:**
```

Comment:
are these really optional, since if the names don't match the code won't run?

Reply:
If they are following the tutorial, then these variables are optional since the defaults will match the tutorial values.